client/main/notification.js

```diff
@@ -0,0 +1,94 @@
+
+'use strict';
+
+const _ = require('underscore');
+var $ = require('jquery');
+var Backbone = require('backbone');
+Backbone.$ = $;
+var SilkyView = require('./view');
+
+var Notification = SilkyView.extend({
+    className: "notification",
+    events : {
+        'click .silky-notification-link'    : '_linkClicked'
+    },
+    initialize: function() {
+        this.model.on("change", this.render, this);
+        this.model.on("change:index", this.movedown, this);
+        this.bottom = (10*(this.model.attributes.index+1)+50*this.model.attributes.index);
```

Comment:
i think spaces around arithmetic operators are probably a good idea. the python linter certainly complains if we don't have them, so might be good to have them for consistency.